src/main/drivers/serial_uart_impl.h

```diff
@@ -29,53 +29,77 @@
 #define UART_RX_BUFFER_SIZE     128
 #endif
 #ifndef UART_TX_BUFFER_SIZE
+#ifdef USE_MSP_DISPLAYPORT
+#define UART_TX_BUFFER_SIZE     1280
```

Comment:
Have you tested building a F1 target with this?

Reply:
No,I didn't have a F1 target

Reply:
I will remove it